index.html

```diff
@@ -1,38 +1,55 @@
 <!DOCTYPE html>
-<html dir="ltr" lang="en-US">
+<html dir="ltr" lang="en">
   <head>
-    <meta content="text/html; charset=utf-8" http-equiv="content-type">
+    <meta charset="utf-8">
     <title>Language Tags and Locale Identifiers for the World Wide Web</title>
     <script src="https://www.w3.org/Tools/respec/respec-w3c-common" async="" class="remove"></script>
     <script class="remove">
       var respecConfig = {
           specStatus: "ED",
-	        edDraftURI: "https://w3c.github.io/ltli/",
+	      edDraftURI: "https://w3c.github.io/ltli/",
           shortName:  "ltli",
           editors: [
                 {   name:       "Addison Phillips",
-                    company:    "Invited Expert"
-	    },
-	              {   name: "Felix Sasaki",
-		                company: "W3C"
-            }
+                    company:    "Amazon.com"
+	            },
+	            {   name: "Felix Sasaki",
+		            company: "Invited Expert"
+                }
           ],
           // if you wish the publication date to be other than today, set this
           //publishDate:  "2015-04-23",
 		  noRecTrack: true,
 	
           previousMaturity: "WD",
-          previousPublishDate:  "2006-06-12",
+          previousPublishDate:  "2015-04-23",
           wg:           "Internationalization Working Group",
           wgURI:        "https://www.w3.org/International/core/",
           wgPublicList: "www-international",
           wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/32113/status",
           github: "w3c/ltli",
 
-localBiblio:  {
+    localBiblio:  {
+
+	"ECMA402": {
```

Comment:
You can use `[ECMA-402]` to refer to ECMA-402 in ReSpec (without adding a custom entry in `localBiblio`).

Reply:
Good catch. Fixed.